data/src/languages.json

```diff
@@ -449,103 +631,157 @@
         "iso639_name": "Gulf Arabic",
         "wiktionary_name": "Gulf Arabic",
         "wiktionary_code": "afb",
-        "casefold": false
+        "casefold": false,
+        "script": {
+            "arab": "Arabic"
+        }
     },
     "hts": {
         "iso639_name": "Hadza",
         "wiktionary_name": "Hadza",
         "wiktionary_code": "hts",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin"
+        }
     },
     "haw": {
         "iso639_name": "Hawaiian",
         "wiktionary_name": "Hawaiian",
         "wiktionary_code": "haw",
-        "casefold": true
+        "casefold": true,
+        "script": {
+            "latn": "Latin",
+            "zyyy": "Common"
```

Comment:
Same as my previous comment: May need to be folded at the implementation level...